### PR TITLE
[auto-bump][chart] kubernetes-dashboard-4.0.3

### PR DESCRIPTION
**Pull request by d2iq-dispatch**
https://github.com/kubernetes/dashboard/tree/master/aio/deploy/helm-chart/kubernetes-dashboard#version-4xx

##### Add Release Notes or "None":

```release-note
Starting from version 4.0.0 of this chart, it will only support Helm 3 and remove the support for Helm 2
```
This is automated bump triggered from the source repo containing the updated Chart/Addon.

### CODE REVIEW

**Review by joejulian — APPROVED** [review]

File: addons/dashboard/dashboard.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: dashboard
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "3.0.2-0"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "3.0.2-1"
     appversion.kubeaddons.mesosphere.io/dashboard: "3.0.2"
```
Comment (s12chung) [inline addons/dashboard/dashboard.yaml:9]: feel like the app version should change....

Comment (joejulian) [reply addons/dashboard/dashboard.yaml:9]: Agreed. The app version seems to actually be `2.2.0`.

**Review by joejulian — APPROVED** [review]

**Comment by s12chung** [comment]
so we're going to not match the `app-version` with the `addon-revision` anyone? if so, i'll ✅ . i dunno the status of that.

**Comment by joejulian** [comment]
I've proposed, many times, that matching a version of the Addon definition to anything but the version of the addon file is problematic at best. Now, because of the error with the app version, the only way to make them match is to go backwards with the revision which sounds like a worse idea and will make the linter fail.

**Comment by s12chung** [comment]
whichever direction we go, as long as there is alignment and consistency, we'll go. it's just the consistency weirdness that holds me back.

**Comment by joejulian** [comment]
@s12chung Who else would you like to see aligned here?

**Comment by s12chung** [comment]
@joejulian you ok bringing it up in the next standup? i'll make a reminder for myself.

**Comment by joejulian** [comment]
sure

**Review by s12chung — APPROVED** [review]
as confirmed during standup

**Comment by joejulian** [comment]
@mesosphere-mergebot backport release/3 release/4

**Pull request merged**